.github/workflows/on-push.yml

```diff
@@ -284,7 +284,7 @@ jobs:
           cd content
           rm -rf .pre-commit-config_template.yaml
           rm -f Tests/docker_native_image_config.json
-          demisto-sdk pre-commit --validate --show-diff-on-failure --verbose
+          demisto-sdk pre-commit --no-validate --show-diff-on-failure --verbose
```

Comment:
why we changed --validate  to --no-validate

Reply:
previously this was used to check that pre-commit will run after some changes in the code. This made PA114 fail successfully because the pack metadata version was not updated. Since I started it seems someone fixed the issue and validate now only runs a specific validation to see that it is triggered (and PA114 is skipped).

Changed it back to the way it works now on master.